foundation/org.eclipse.persistence.corba/build.properties

```diff
@@ -0,0 +1,5 @@
+bin.includes = .
+source.. = src/main/java,\
+           src/main/resource/
+src.includes = src/main/java/,\
+               META-INF/
```

Comment:
Directories specified there are Maven compatible, but non consistent with source/resource directories in other modules like _org.eclipse.persistence.core_, _org.eclipse.persistence.extension_. `src/main/resource/` is not used.

Reply:
does it make sense to be consistent on places known to be wrong and/or changed in the near future?